src/core/template-host.ts

```diff
@@ -58,16 +58,22 @@ class TemplateHost {
         const ownerName = meta.ownerName;
 
         for (const tmpl of templateProps) {
+            let propName = props[tmpl.component] ? tmpl.component : tmpl.render;
+            let propsGetter = meta.propsGetter;
             if (!props[tmpl.component] && !props[tmpl.render]) {
-                continue;
+                if (meta.useChildren(tmpl.tmplOption)) {
+                    propName = "children";
+                    propsGetter = () => () => meta.propsGetter("children");
```

Comment:
Does reassigning `propsGetter` really make sense after changing `propName` to `"children"` ?